src/mal/SHCore.cpp

```diff
@@ -433,12 +433,18 @@ class malSHMesh : public malValue, public malRoot {
 
   SHMesh *value() const { return m_mesh.get(); }
 
-  void schedule(malSHWire *wire) {
+  void schedule(malSHWire *wire, bool compose = true) {
     auto cp = SHWire::sharedFromRef(wire->value());
-    m_mesh->schedule(cp);
+    m_mesh->schedule(cp, Var::Empty, compose);
```

Comment:
Can we remove compose from schedule instead so the caller can decide by either calling:
```
compose(...)
schedule(...)
```
or just
```
schedule(...)
```
?

Reply:
yeah but what about edn code? keep schedule abstraction right?

Reply:
Yeah edn could stay the same for now I think, just talking about the cpp code

Reply:
If edn turns into (shedule .. true) or false for compose, it'd split it there too.

Reply:
Leaving this out as requires a bigger refactor that is a waste of time for me sadly @guusw

Reply:
https://github.com/fragcolor-xyz/shards/issues/526